src/sentry/api/endpoints/organization_details.py

```diff
@@ -116,6 +123,48 @@ def update_organization_scenario(runner):
         )
 
 
+class TrustedRelaySerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        return convert_dict_key_case(instance, snake_to_camel_case)
+
+    def to_internal_value(self, data):
+
+        key_name = "{unknown}"
+        try:
+            key_name = data.get(u"name")
+            public_key = data.get(u"publicKey")
+
+            PublicKey.parse(public_key)
```

Comment:
If you like, you can check for `not public_key` before and issue a specific error message that the key should not be empty.

Reply:
Code refactored to create detailed error messages.

I check `publicKey` after `name` in the hope that I get a valid name that I can use in the error message so I can hint which `publicKey` has a problem.